python/ray/_private/tracing_utils.py

```diff
@@ -0,0 +1,72 @@
+import inspect
+import os
+from typing import Callable, List
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter, SimpleSpanProcessor
+from ray._private.utils import import_attr
+from ray.serve.schema import TracingConfig
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+
+
+DEFAULT_TRACING_EXPORTER_IMPORT_PATH = (
+    "ray._private.tracing_utils:default_tracing_exporter"
+)
+
+
+def default_tracing_exporter() -> List[SimpleSpanProcessor]:
+    os.makedirs("/tmp/spans", exist_ok=True)
+    return [
+        SimpleSpanProcessor(
+            ConsoleSpanExporter(out=open(f"/tmp/spans/{os.getpid()}.json", "a"))
+        )
+    ]
```

Comment:
is the plan to export spans to disk by default? if so, this should be scoped to the "session dir" (`/tmp/ray/session_.../tracing_spans`)